simtools/db/db_handler.py

```diff
@@ -1039,8 +1039,14 @@ def add_new_parameter(
             db_entry["instrument"] = names.validate_telescope_name(telescope)
         elif "sites" in collection_name:
             db_entry["instrument"] = names.validate_site_name(site)
+        # TODO: generalize later
```

Comment:
Suggest to remove this comment and open an issue regarding the generalization.

Reply:
Removed, opening an issue.

Reply:
Issue is here: #871